packages/clarity-js/src/interaction/pointer.ts

```diff
@@ -35,7 +35,7 @@ export function observe(root: Node): void {
 function mouse(event: Event, root: Node, evt: MouseEvent): void {
     mouse.dn = FunctionNames.PointerMouse;
     let frame = iframe(root);
-    let d = frame ? frame.contentDocument.documentElement : document.documentElement;
+    let d = frame && frame.contentDocument ? frame.contentDocument.documentElement : document.documentElement;
```

Comment:
same here
frame.contentDocument shouldnt be null here. iframe(root) returns the frame based on root -> the root is the contentDocument in this case. frame should be null if frame.contentDocument is null.